exercises/dominik_arominski/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+std::string RemovePunctuation(std::string word) {
+  const char punctuation[] = {'.', ',', '?', '!', '"', '\'', ')', '('};
+  for (auto punc : punctuation) {
+    if(word.find(punc) < 100) {
```

Comment:
Can you explain what's the reason for this check?

Reply:
It was meant to find whether the punctuation character was in the string. I believe it would be better if the check was: if(word.find(punc) != std::string::npos).